### ISSUE TITLE
Improve Skill Search Feedback

### ISSUE THREAD

**Issue by KramNamez** [puzzle/skills#776]
I was trying to see if someone has a specific skill (API Management, in this case). The lack of feedback on the search page left me wondering if I needed to manually test all possible combinations of skill level and interest.

I _hope_ it searches anything with skill level x or greater and interest y or greater, but I wouldn't know...

And there was no explicit search button or feedback to tell me that it was actually searching.

**Comment by KramNamez** [comment]
Right, but it says "no results" already _before_ I start searching. I know that that's because technically it searched for "nothing" already, but that's not how I interact with the site.

So I input the skill I'm searching for and... nothing changes, nothing happens. If it could default to something like "please input a skill to search for" and then switched to "no results" once I select my skill, that'd be much clearer. I did something, it responds, instead of me doing something and not knowing if nothing changed because that's correct, or because it failed to search.

It would probably be even better if it said "no results for skill X at level y or greater with interest z or greater" - that would immediately also explain how those parameters are treated.

Since skill X would then change on my input, I would also see that it is working and didn't silently break.

**Comment by Robin481** [comment]
Sure, that sounds very reasonable! Thanks for your feedback.
Will do this sometime early next year :v: 

TODO:
- Display "Add a Skill to search" message when first opening the skill search
- Display "no result for skill at level X or greater with interest Z or greater" whenever a search with no results is completed
- Figure out what to display for a multi skill search. Maybe just list all the skills as well :thinking: 
- Ask someone from UX for further input or if this should be fine now
- Write some tests

**Comment by nevio18324** [comment]
### Stand 18.2 ###
Das feedback in der skill search wurde verbessert das ganze muss nur noch
- 
- Mit dem Ux abgeklärt werden
- Das neue feedback testen

**Comment by nevio18324** [comment]
### Stand 27.2 ###
Ich habe die review changes umgesetzt der derzeitige satz hat jedoch noch schlechtes deutsch und
die checks failen
